### PR TITLE
Added scoped module support

### PR DESCRIPTION
**Pull request by TatumCreative**
From #104 this adds scoped module support. Please review and let me know if there is anything I need to change as this code-base was all new to me, but hopefully I got everything. The biggest question-mark on this PR is that I had to bump the npm version to support scoped modules.

### CODE REVIEW

**Comment by jfhbrook** [comment]
Jesus, this has been using npm 1?? Wow. I still use npm 2, but maybe npm 3 is better for browserify builds?

**Comment by jfhbrook** [comment]
FTR npm 1 doesn't support scoped modules, but npm 2+ does.

File: singular.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ module.exports = function (app, bundle) {
   //
   // Singular bundles
   //
-  app.get('/bundle/:module', singular(bundle));
-  app.get('/debug-bundle/:module', singular(bundle, { debug: true }));
-  app.get('/standalone/:module', singular(bundle, { standalone: true }));
-  app.get('/debug-standalone/:module', singular(bundle, { standalone: true, debug: true }));
+  app.get('/bundle/:scope?/:module', singular(bundle));
```
Comment (jfhbrook) [inline singular.js]: Oh neat, I've actually never written a collapsing parameter like that.

But maybe we should go for @scope%2Fmodule ?

**Comment by jfhbrook** [comment]
@maxogden what do you think? I think npm 3 is fine but that we should encode the slash in the package name.

@TatumCreative this is great, thank you.

**Comment by maxogden** [comment]
I'm personally -1 on npm 3, I'd stick with whatever npm ships with the LTS version of node (2.x currently) just to minimize potential bugs

**Comment by TatumCreative** [comment]
Ok, I updated the forward slash to a %2F and did npm down to v2.14.15. Let me update the docs real quick, so that people know to do the %2F for scoped modules. Let me know if there is anything else!

**Pull request merged**